src/tools/gophish_import.py

```diff
@@ -117,23 +116,24 @@ def load_groups(api, assessment):
                 break
             except Error as e:
                 if e.message == "Group name already in use":
-                    logging.warning(f"{e}. Finding previously loaded group to delete.")
+                    logging.warning("%s. Finding previously loaded group to delete.", e)
                     groups = api.groups.get()
                     logging.debug(
-                        f"Checking {len(groups)} for previously imported group to get ID"
+                        "Checking %i for previously imported group to get ID",
+                        len(groups),
                     )
                     for old_group in groups:
                         if old_group.name == new_group.name:
-                            logging.debug(f"Deleting Group with ID {old_group.id}")
+                            logging.debug("Deleting Group with ID %i", old_group.id)
                             api.groups.delete(old_group.id)
                             logging.info("Re-Loading new group.")
                 else:
-                    logging.error(f"{e}\n")
+                    logging.error("%s", e)
```

Comment:
Why not use [`logging.exception()`](https://docs.python.org/3/library/logging.html#logging.Logger.exception) here instead of recreating the wheel?

Reply:
See 44e24edb9ac96eb42759bdea8a9229563811ca59

Reply:
Also been updated here 3ee423033fefa9873fd13190c0e6edb575cd0080